docs/DeveloperGuide/Tutorials/AddV2I/AddV2I/index.md

```diff
@@ -0,0 +1,24 @@
+# Add V2I
+
+## 1. Add V2I prefab
+
+![add prefab](add_prefab.gif)
+
+## 2. Select EGO transform
+   
+![select ego transform 1](select_ego_transform_1.png)
+![select ego transform 2](select_ego_transform_2.gif)
+![select ego transform 3](select_ego_transform_3.png)
+
+## 3. Configure
```

Comment:
To raczej powinno byc w UserGuide, zachecam do dodania `Components->V2I `opisu co to jest za co odpowiada.